azurerm/resource_arm_analysis_services_server.go

```diff
@@ -208,6 +220,14 @@ func resourceArmAnalysisServicesServerRead(d *schema.ResourceData, meta interfac
 		}
 
 		d.Set("querypool_connection_mode", string(serverProps.QuerypoolConnectionMode))
+
+		if serverFullName := serverProps.ServerFullName; serverFullName != nil {
+			d.Set("server_full_name", serverProps.ServerFullName)
+		}
```

Comment:
since this is a `*string` we can make this:

```suggestion
	d.Set("server_full_name", serverProps.ServerFullName)
```

as d.Set will take care of de-referencing the pointered object